samples/session-replay-mobile-schema/session-replay/incremental-snapshot-record.json

```diff
@@ -0,0 +1,53 @@
+{
+    "timestamp": 1657620232337,
+    "type": 3,
+    "data": [
+        {
+            "source": 1,
```

Comment:
Ok so regarding that, the version I have from the browser sdk does not match:

```
export enum IncrementalSource {
    Mutation = 0,
    MouseMove = 1,
    MouseInteraction = 2,
    Scroll = 3,
    ViewportResize = 4,
    Input = 5,
    TouchMove = 6,
    MediaInteraction = 7,
    StyleSheetRule = 8
}
```

it should be source=1 isn't it ?